### PR TITLE
FB Channel

### PR DESCRIPTION
**Pull request by nicpottier**
@norkans7 this isn't done yet but would be a good next channel to port over. (viber being the next)

### CODE REVIEW

**Comment by nicpottier** [comment]
Actually looking at this it is pretty far off, so I'll take care of it (there's some weirdness around FB referral ids which needs working out).

@norkans7 why don't you do viber next?

**Comment by johncordeiro** [comment]
What's happening on facebook referral ids? Maybe we can help with that.

**Comment by nicpottier** [comment]
We're working on it, thanks though!

File: handlers/facebook/facebook.go [hunk 1]
```diff
@@ -1,6 +1,386 @@
...
+func init() {
+	courier.RegisterHandler(NewHandler())
+}
+
+type handler struct {
+	handlers.BaseHandler
+}
+
+const (
+	maxLength = 640 // Facebook API says 640 is max for the body
```
Comment (norkans7) [inline handlers/facebook/facebook.go]: this const can join the other on top

**Pull request merged**